ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/system-control-plane.j2

```diff
@@ -0,0 +1,34 @@
+{# eos - system #}
+{% if system.control_plane is arista.avd.defined %}
+!
+system control-plane
+{#     control_plane tcp mss ceiling #}
+{%     if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+{%         set cp_mss_cli = "tcp mss ceiling " %}
```

Comment:
The current code will produce a trailing space if you only define ipv4. We normally solve this by moving the spaces to the beginning of the optional cli:
Remove trailing space from line 7.
Add space before `ipv4` in line 9 and before `ipv6` in line 12.
Remove concatenated `" "` in line 9.

Reply:
Updated in 7f64921e53cc2146c20a104a9fb2105c2ad1d79d to remove possibility of trailing whitespace